modules/sdk-coin-atom/src/lib/utils.ts

```diff
@@ -166,13 +193,21 @@ export class Utils implements BaseUtils {
     return encodedMessage.typeUrl;
   }
 
-  getGasBudgetFromDecodedTx(decodedTx: DecodedTxRaw): GasFeeLimitData {
+  getGasBudgetFromDecodedTx(decodedTx: DecodedTxRaw): FeeData {
     return {
       amount: decodedTx.authInfo.fee?.amount as Coin[],
-      gas: Number(decodedTx.authInfo.fee?.gasLimit),
+      gasLimit: Number(decodedTx.authInfo.fee?.gasLimit),
     };
   }
 
+  getPublicKeyFromDecodedTx(decodedTx: DecodedTxRaw): string | undefined {
```

Comment:
Kindly add jsdocs, lets be consistent with rest of the functions, Kindly add to all functions i the file which doesn't have it

Reply:
done